homeassistant/components/telegram/notify.py

```diff
@@ -76,6 +77,11 @@ def send_message(self, message="", **kwargs):
             parse_mode = data.get(ATTR_PARSER)
             service_data.update({ATTR_PARSER: parse_mode})
 
+        # Set disable_web_page_preview
+        if data is not None and ATTR_DISABLE_WEB_PREV in data:
+            disable_web_page_preview = data.get(ATTR_DISABLE_WEB_PREV)
```

Comment:
```suggestion
            disable_web_page_preview = data[ATTR_DISABLE_WEB_PREV]
```

Reply:
Please apply this change suggestion. The old code for the other fields is sloppy and uses `dict.get` when it isn't needed, but new code should be good.

Reply:
Shall we correct a code for ATTR_PARSER, ATTR_DISABLE_NOTIF & ATTR_MESSAGE_TAG too?

Reply:
Not in this PR. You're welcome to clean that up in a separate PR. 👍